docs/install-configuration.md

```diff
@@ -0,0 +1,175 @@
+# Configuration
+
+A YAML configuration file configures many aspects of how your Nakama server runs. You can run Nakama without specifying a configuration file and rely on the the default settings instead.
+
+## Specifying a config file
+You can specify a configuration file at run-time using --config flag.
+
+```shell fct_label="Shell"
+nakama --config my-special-config.yml
+```
+
+If you are running Nakama via Docker-Compose, you'll need to bind a folder in your machine so that it's available in Docker. Follow this guide to [setup folder binding](install-docker-quickstart.md#data).
+
+## Common properties
+
+There are a few configuration properties that need to be changed in most environments. The full list of configurations is at the [bottom of the page](#server-configuration).
+
+| Parameter                | Description
+| ---------                | -----------
+| `name`                   | Nakama node name (must be unique) - It will default to `nakama-xxxx` where `xxxx` is 4 random characters.
+| `data_dir`               | An absolute path to a writeable folder where Nakama will store its data, including logs. Default value is the working directory that Nakama was started on.
+| `runtime.path`           | Path of modules to scan and load. Defaults to `data_dir/modules`.
+| `database.address`       | List of database nodes to connect to. It should follow the form of `username:password@address:port/dbname` (`postgres://` protocol is appended to the path automatically). Defaults to `root@localhost:26257`.
+| `socket.server_key`      | Server key to use to establish a connection to the server. Default value is `defaultkey`.
+| `session.encryption_key` | The encryption key used to produce the client token. Default value is `defaultencryptionkey`.
+| `runtime.http_key`       | Key is used to protect the server's runtime HTTP invocations. Default value is `defaultkey`.
+
+!!! warning "Production settings"
+    You must change the values of **`socket.server_key`**, **`session.encryption_key`** and **`runtime.http_key`** before you deploy Nakama to a live production environment. 
+
+    Follow the [production settings deployment guide](deployment-production-settings.md) for more information.
+
+## Server Configuration
+
+Nakama has various configuration options to make it as flexible as possible for various use cases and deployment environments.
+
+Nakama ships with sane default values for all config options, therefore you'll only need to override a subset of the options. You can also setup your own config file, and furthermore override the values in the config file via command-line flags. For instance, to override Runtime Path:
```

Comment:
"sane default files" - is 'sane' a common term here?  otherwise:    "Nakama ships with common sense default values as selected by our engineers for all config options..." ?